helpers/handlebars/middleware.js

```diff
@@ -94,6 +94,7 @@ module.exports = (req, res, next) => {
 		testId: 'Meine Dateien',
 		icon: 'folder-open',
 		link: '/files/',
+		excludedPermission: 'COLLABORATIVE_STORAGE',
```

Comment:
for dev environments it might make sense to be able to see both file areas at the same time

Reply:
Yes, I was thinking too. But I thought for dev you could just take two users instead of introducing a short-lived permission.
The next ticket is to write a migration script to add this permission to a role like admin: https://ticketsystem.dbildungscloud.de/browse/N21-401